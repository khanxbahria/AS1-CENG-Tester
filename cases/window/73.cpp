#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->changeActiveTabTo(4);
Tab K = Tab("kk", "hh","y");
wind->moveActiveTabTo(4);
Tab F = Tab("kk", "kk","s");
wind->moveActiveTabTo(5);
wind->newTab(F);
Tab P = Tab("ggg", "k","rr");
wind->moveActiveTabTo(0);
Tab A = Tab("rrr", "xx","jjj");
wind->closeTab();
wind->print();
Tab R = Tab("mmm", "fff","kkk");
wind->changeActiveTabTo(4);
wind->getActiveTab();
wind->isEmpty();
Tab B = Tab("ooo", "nnn","y");
wind->moveActiveTabTo(3);
Tab U = Tab("ppp", "gg","www");
wind->getActiveTab();
Tab G = Tab("ccc", "i","dd");
wind->print();
Tab N = Tab("zz", "ddd","vv");
wind->moveActiveTabTo(3);
Tab C = Tab("uu", "ll","m");
wind->isEmpty();
Tab X = Tab("cc", "jj","a");
wind->print();
Tab W = Tab("a", "y","w");
wind->print();
Tab D = Tab("a", "vv","yy");
wind->changeActiveTabTo(5);
Tab T = Tab("pp", "jjj","m");
wind->isEmpty();
wind->changeActiveTabTo(0);
Tab Q = Tab("c", "ss","ee");
wind->isEmpty();
wind->moveActiveTabTo(3);
wind->newTab(A);
wind->moveActiveTabTo(2);
wind->closeTab();
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(3);
wind->closeTab();
wind->getActiveTab();
wind->print();
wind->moveActiveTabTo(3);
wind->moveActiveTabTo(0);
wind->getActiveTab();
wind->closeTab();
Tab I = Tab("hh", "g","t");
wind->moveActiveTabTo(0);
wind->print();
wind->getActiveTab();
wind->changeActiveTabTo(4);
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(3);
wind->moveActiveTabTo(5);
wind->newTab(B);
Tab J = Tab("rrr", "r","oo");
wind->closeTab();
wind->newTab(C);
Tab E = Tab("b", "nnn","nn");
wind->moveActiveTabTo(0);
wind->getActiveTab();
wind->isEmpty();
Tab S = Tab("hh", "ppp","r");
wind->getActiveTab();
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(2);
wind->getActiveTab();
wind->print();
wind->getActiveTab();
Tab Z = Tab("ddd", "nn","oo");
wind->changeActiveTabTo(0);
wind->isEmpty();
wind->newTab(D);
Tab H = Tab("ttt", "h","jj");
wind->print();
wind->newTab(E);
wind->newTab(G);
wind->isEmpty();
Tab V = Tab("yyy", "h","h");
wind->moveActiveTabTo(1);
wind->newTab(H);
wind->newTab(I);
wind->print();
wind->closeTab();
wind->newTab(J);
wind->changeActiveTabTo(5);
Tab Y = Tab("qq", "z","p");
wind->newTab(K);
Tab O = Tab("qqq", "mmm","xx");
wind->getActiveTab();
wind->changeActiveTabTo(2);
wind->isEmpty();
wind->changeActiveTabTo(0);
wind->closeTab();
Tab L = Tab("m", "ccc","f");
wind->changeActiveTabTo(3);
wind->newTab(L);
wind->getActiveTab();
wind->newTab(N);
wind->changeActiveTabTo(1);
wind->changeActiveTabTo(1);
wind->changeActiveTabTo(3);
wind->newTab(O);
wind->changeActiveTabTo(3);
wind->getActiveTab();
wind->isEmpty();
wind->moveActiveTabTo(4);
wind->isEmpty();
wind->closeTab();
wind->print();
wind->isEmpty();
wind->newTab(P);
wind->changeActiveTabTo(5);
wind->closeTab();
wind->changeActiveTabTo(3);
wind->moveActiveTabTo(4);
wind->closeTab();
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(4);
wind->moveActiveTabTo(1);
delete wind;
}
	return 0;
}