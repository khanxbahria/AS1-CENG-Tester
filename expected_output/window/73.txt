The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
ll
uu
m

The active tab is tab 1
ll
uu
m

vv
a
yy

The active tab is tab 3
ll
uu
m

i
ccc
dd

h
ttt
jj

g
hh
t

vv
a
yy

nnn
b
nn

The active tab is tab 4
i
ccc
dd

h
ttt
jj

vv
a
yy

mmm
qqq
xx

ccc
m
f

ddd
zz
vv

nnn
b
nn

hh
kk
y

